src/app.ts

```diff
@@ -111,10 +141,18 @@ async function app() {
   logger.info('Converting FSH to FHIR resources...');
   const outPackage = exportFHIR(tank, defs);
 
-  fs.ensureDirSync(program.out);
+  let outDir = program.out;
+  if (isIgPubContext && !program.out) {
+    // When running in an IG Publisher context, default output is the parent folder of the tank
+    outDir = path.join(input, '..');
+  } else if (!program.out) {
+    // Any other time, default output is just to 'build'
+    outDir = path.join('.', 'build');
+  }
```

Comment:
I think in the cases above where we use a default output path (instead of provided, maybe we should log that?)

Reply:
That seems reasonable. I'll add it